### PR TITLE
Comment Reply: update fullscreen UI

### PR DESCRIPTION
**Pull request by ScoutHarris**
Ref: #17412 

This updates the UI for the fullscreen Reply view.

Notes:
- There is no placeholder text yet.

To test:
- Go to any of these views as they all use the same Reply view:
  - My Site > old Comment details.
  - My Site > new Comment details.
  - Notifications > Comments > notification details.
  - Reader > post comments > reply to post or comment.
- Tap the up arrow to enter fullscreen reply.
- Verify the nav title does not appear.
- Verify the nav bar bottom line does not appear.
- Verify the `Reply` button is disabled.
- Enter some text. Verify the `Reply` button is enabled.

| Simulator Screen Shot - iPhone 12 Pro Max - 2021-11-10 at 13 12 29 | Simulator Screen Shot - iPhone 12 Pro Max - 2021-11-10 at 13 12 34 |
|--------|-------|

## Regression Notes
1. Potential unintended areas of impact
Should be none.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Verified the UI looked and functioned correctly.

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Comments/FullScreenCommentReplyViewController.swift [hunk 1]
```diff
@@ -14,43 +14,33 @@ fileprivate enum SuggestionsPosition: Int {
...
-        configureAppearance()
+        configureNavigationAppearance()
     }
 
     public override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
-
-        enableRefreshButtonIfNeeded(animated: false)
-
+        refreshReplyButton()
```
Comment (ScoutHarris) [inline WordPress/Classes/ViewRelated/Comments/FullScreenCommentReplyViewController.swift:43]: I didn't realize this was in `viewDidAppear`. I moved it to `viewWillAppear` in #17448.

**Comment by ScoutHarris** [comment]
I did notice that sometimes when going from minimized to fullscreen and back to minimized, the minimized textView is a bit cut off at the bottom. That's a legacy issue. And we know what I do with those! 😄  I'll look at it separately.

**Review by dvdchr — APPROVED** [review]
LGTM! :shipit:

**Comment by dvdchr** [comment]
Not sure if this is intended or not, but in dark color scheme, the Reply button has a white color when active. For comparison, the old fullscreen reply view had a blue tint color for the paper airplane icon.

Screen Shot 2021-11-11 at 16 36 44

Feel free to ignore if this is expected! 😄

**Comment by ScoutHarris** [comment]
> in dark color scheme, the Reply button has a white color when active

I've attempted to change the colors, but for some reason they are not taking effect. I'll have to look at this separately so I can dig in and figure out what's going on.

**Pull request merged**